packages/frames.js/src/core/types.ts

```diff
@@ -293,6 +310,8 @@ export type FramesOptions<
   /**
    * If provided the state will be signed with this secret and validated on subsequent requests.
    * If the signature is not valid, error is thrown.
+   *
+   * Also used to sign image rendering payloads.
```

Comment:
This is little bit weird, I would use this value as fallback value for some `imageSecret` or something like that. Or rename the `stateSigningSecret` to `signingSecret` since it is used to sign more than state.

Reply:
you're right. i also have mixed feelings about this. even using it as a fallback might not be intuitive. will split them out for now i think. don't want to rename stateSigningSecret because that would be a breaking change. alternatively we could add `signingSecret` which can be overridden by `stateSigningSecret` and images `secret`?

Reply:
that's great idea!